### PR TITLE
RDoc-1362 v4.2 Documentation project

### PR DESCRIPTION
**Pull request by talweiss1982**
Please merge this with high priority otherwise this PR is going to conflict with everybody work.
We can make modification later if needed.

### CODE REVIEW

**Comment by ppekrol** [comment]
@topalach please review

File: Raven.Documentation.Parser/Compilation/ToC/TableOfContentsCompiler.cs [hunk 1]
```diff
@@ -95,7 +95,10 @@ private IEnumerable<CompilationResult> GenerateTableOfContentItems(string direct
                 {
                     tableOfContentsItem.Languages = GetLanguagesForTableOfContentsItem(directory, item.Name).ToList();
                     tableOfContentsItem.AddSupportedVersions(item.SupportedVersions);
+                    if (item.Description == "Getting Started")
```
Comment (topalach) [inline Raven.Documentation.Parser/Compilation/ToC/TableOfContentsCompiler.cs]: Perhaps we don't need this `if` block.

Comment (topalach) [reply Raven.Documentation.Parser/Compilation/ToC/TableOfContentsCompiler.cs]: The rest of the changes are fine.

Comment (talweiss1982) [reply Raven.Documentation.Parser/Compilation/ToC/TableOfContentsCompiler.cs]: yes this is leftover from debugging...

**Pull request merged**